cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -25,7 +25,6 @@ spec:
     metadata:
       labels:
```

Comment:
@mikedanese, I noticed your comment "need a version label or two deployments will select the same set of replicasets". This is news to me, but I'm a novice so am probably confused. Could you spare a moment explain this concern in more detail, or point me to some relevant docs? Thanks.